### DISCUSSION TITLE
Could not load your colorscheme

### DISCUSSION THREAD

**Discussion by davidgao7** [LazyVim/LazyVim#3883] [Q&A] [answered]
Hello everyone,
whe I run sync using lazy 5min ago, tokyonight cannot load

```lua
Could not load your colorscheme

.../share/nvim/lazy/tokyonight.nvim/lua/tokyonight/util.lua:36: attempt to perform arithmetic on a nil value

# stacktrace:
  - /tokyonight.nvim/lua/tokyonight/util.lua:36 _in_ **blendChannel**
  - /tokyonight.nvim/lua/tokyonight/util.lua:40 _in_ **blend**
  - /tokyonight.nvim/lua/tokyonight/groups/neo-tree.lua:7 _in_ **get**
  - /tokyonight.nvim/lua/tokyonight/groups/init.lua:144 _in_ **load**
  - /tokyonight.nvim/lua/tokyonight/theme.lua:15 _in_ **load**
  - /LazyVim/lua/lazyvim/config/init.lua:14 _in_ **colorscheme**
  - /LazyVim/lua/lazyvim/config/init.lua:210
  - /LazyVim/lua/lazyvim/config/init.lua:208 _in_ **setup**
  - /LazyVim/lua/lazyvim/init.lua:7 _in_ **setup**
  - ~/.config/nvim/lua/config/lazy.lua:9
  - ~/.config/nvim/init.lua:2
```
<img width="874" alt="Screenshot 2024-07-02 at 9 27 22 AM" src="https://github.com/LazyVim/LazyVim/assets/39502459/ae104764-3262-4b59-b950-1b8efb3068ff">

**Comment by dpetka2001** [comment]
https://github.com/folke/tokyonight.nvim/blob/5b1e0e9545f9e619427657b8d5569039c8c51e2c/NEWS.md?plain=1#L18-L19

`sidebars` has been removed as an option from latest Tokyonight. I'm not aware how you can achieve transparency for Neo-tree and other sidebars at the moment. Maybe ask at the Tokyonight repo or ping the maintainer himself?

**Reply by folke** [reply]
No, not, that option still works.
Was just a bug in something I added to neo-tree. All fixed now! :)

**Reply by davidgao7** [reply]
thanks for the reply, I commented the sidebars option, I don't use neo-tree often after discovering harpoon

**Comment by folke** [comment, marked as answer]
Fixed...

**Reply by davidgao7** [reply]
sorry man
